diff --git a/6502_functional_test.ca65 b/6502_functional_test.ca65
index 314b760..c40732a 100644
--- a/6502_functional_test.ca65
+++ b/6502_functional_test.ca65
@@ -131,7 +131,7 @@ disable_selfmod = 0

 ;report errors through I/O channel (0=use standard self trap loops, 1=include
 ;report.i65 as I/O channel, add 3.5 kB)
-report = 0
+report = 1

 ;RAM integrity test option. Checks for undesired RAM writes.
 ;set lowest non RAM or RAM mirror address page (-1=disable, 0=64k, $40=16k)
@@ -140,7 +140,7 @@ ram_top = -1

 ;disable test decimal mode ADC & SBC, 0=enable, 1=disable,
 ;2=disable including decimal flag in processor status
-disable_decimal = 0
+disable_decimal = 1

 ;macros for error & success traps to allow user modification
 ;example:
@@ -5924,7 +5924,7 @@ nmi_trap:
         trap            ;check stack for conditions at NMI
         jmp start       ;catastrophic error - cannot continue
 res_trap:
-        trap            ;unexpected RESET
+        ;trap           ;emu start from reset vector
         jmp start       ;catastrophic error - cannot continue

         dey
@@ -6010,7 +6010,7 @@ break2:                 ;BRK pass 2
         jmp start       ;catastrophic error - cannot continue

     .if report = 1
-        include "report.i65"
+        .include "report.i65"
     .endif

 ;copy of data to initialize BSS segment
